### ISSUE TITLE
Explore learning rate decay

### ISSUE THREAD

**Issue by jsphon** [jsphon/reinforcement_learning#43]
The model only starts to convert if we have a learning rate of 0.001. But never quite gets to full convergence. Perhaps we need learning rate decay?

**Comment by jsphon** [comment]
We will need to use 2 versions...

One version, with the learning rate calculated in Python, if we are looping over train_op

Another version, with learning rate calculated in tensorflow, if we are using a tensorflow while loop/
